bin/pip.py

```diff
@@ -51,11 +52,19 @@
     'lazy_object_proxy': 'lazy-object-proxy',
 }
 
-SITE_PACKAGES_FOLDER = os.path.expanduser('~/Documents/site-packages')
+_stash = globals()['_stash']
+
+if _stash.PY3:
+    unicode = str
+    str = bytes
```

Comment:
`str = bytes` doesn't seem like a good idea, that will be very confusing for people who expect Python 3 code (where `str` means `unicode`). I think it would be better to always use `unicode` and `bytes` explicitly. That way it's always clear which type of string is meant.

Actually, there is a case where you want to use `str` instead of `bytes`/`unicode`: some strings have to have the "default string" type (bytestring in Python 2 and Unicode string in Python 3), for example the attribute name passed to `getattr`. This is another reason why overwriting `str` would be problematic, because then there's no easy way to use the "default string" type.

Reply:
These are the kind of places where six shines...

Reply:
```python
try:
    unicode        # Python 2
except NameError:
    unicode = str  # Python 3
```